lib/cc/engine/analyzable_files.rb

```diff
@@ -1,39 +1,29 @@
 module CC
   module Engine
     class AnalyzableFiles
-      def initialize(directory, config)
-        @directory = directory
+      def initialize(config)
         @config = config
       end
 
       def all
         @results ||= if @config["include_paths"]
-          build_files_with_inclusions(@config["include_paths"])
+          include_paths = @config["include_paths"]
+          if include_paths == ["./"]
+            ["."]
```

Comment:
I'd also consider pushing this translation closer to where it's actually needed, maybe as part of building the argument to the coffee binary?